conf/crick.config

```diff
@@ -1,6 +1,6 @@
 //Profile config names for nf-core/configs
 params {
-    config_profile_description = 'The Francis Crick Institute CAMP HPC cluster profile provided by nf-core/configs.'
+    config_profile_description = 'The Francis Crick Institute NEMO HPC cluster profile provided by nf-core/configs.'
```

Comment:
I guess too late, but given it has a distinct name: do you have multiple? If so maybe it makes sense to have one config per cluster e.g. `--profile nemo`

Reply:
not too late! But no, the cluster name got changed to nemo from camp, camp is gone